src/chrome/content/rules/Github-Pages.xml

```diff
@@ -5,6 +5,26 @@
 
 	<target host="*.github.io" />
 
+
+	<!-- pages that require http -->
+	<exclusion pattern="^http://duckietv\.github\.io/DuckieTV/" />
```

Comment:
A couple of issues here:

* http://duckietv.github.io/DuckieTV (without the trailing slash) still redirects to the HTTPS version.
* I'm actually seeing some CORS issues on http://duckietv.github.io/DuckieTV/ that may or may not be related:

![skaermbillede fra 2016-06-16 22 47 28](https://cloud.githubusercontent.com/assets/6169306/16132708/9e8b68ce-3414-11e6-974d-9f7deff87c2a.png)